docs/src/Tools.md

```diff
@@ -161,27 +183,27 @@ Explain how the timing of the blinking of the LED is controlled.
 
 ## 3.3 A First Lingua Franca Program
 
-Start code in the root `lf-pico` directory:
+Start `code` in the root of your repository based on `lf-3pi-template` (see [Getting Started](http://localhost:3000/GettingStarted.html#clone-your-repository)):
 
-```
-    cd lf_pico
-    code .
+```bash
+$ cd ~/my-rpi3
+$ code .
 ```
 
 Open and examine the `Blink.lf` program in the `src` directory.  You may want to open the diagram and drag its subwindow to the bottom so that you something like this:
 
 <img src="img/BlinkInCode.png" alt="Blink in code"/>
 
-To compile this program, select View->Terminal from the menu, and type in the terminal (or an external terminal window if you prefer),
+To compile this program, select <kbd>View > Terminal</kbd> from the menu, and type in the terminal (or an external terminal window if you prefer),
```

Comment:
Does the VS Code extension provide some built-in way to run lfc? Or do you have to use the CLI as explained here?

Reply:
a959cac0a6643e3a01dacea745c2d0f788d08955